website/public/locales/en/leaderboard.json

```diff
@@ -35,5 +35,6 @@
   "weekly": "Weekly",
   "xp_progress_message": "You need {{need, number, integer}} more points to reach the next level!",
   "your_account": "Your account",
-  "your_stats": "Your statistics"
+  "your_stats": "Your statistics",
+  "user_stats": "User statistics"
```

Comment:
You dont need to translate text in the admin area

Reply:
removed it